unit_testing/test_register.py

```diff
@@ -167,11 +189,36 @@ def test_register_step_ants_registration(step2_data):
      verbose=cli_params.verbose,
     )
 
-@pytest.mark.skip(reason="TODO")
-def test_register_step_ants_slice_regularized_registration():
-     """
-     """
-     raise NotImplementedError()
+
+def test_register_step_ants_slice_regularized_registration(step_axial_data_in_same_space):
+    """
+    Read CSV file, remove first element, flatten and convert to float
+    """
+    def csv2num(file_csv):
+        with open(file_csv, newline='') as csvfile:
+            spamreader = list(csv.reader(csvfile, delimiter=','))
+        # Remove first element corresponding to header ['Tx', 'Ty']
+        spamreader.pop(0)
+        # Flatten
+        spamreader = [item for sublist in spamreader for item in sublist]
+        # Convert each element to float
+        spamreader = [float(i) for i in spamreader]
+        return spamreader
+
+    src, dest, step, cli_params = step_axial_data_in_same_space
+
+    warp_forward_out, warp_inverse_out = register_step_ants_slice_regularized_registration(
+        src=src,
+        dest=dest,
+        step=step,
+        metricSize='4')
+
+    # Verify integrity of the output Tx Ty file.
+    txty_result = csv2num('step1TxTy_poly.csv')
+    txty_groundtruth = csv2num(
+        os.path.join(__sct_dir__, 'spinalcordtoolbox', 'testing', 'step1TxTy_poly_groundtruth.csv'))
```

Comment:
We have this:

```suggestion
        sct_test_path('step1TxTy_poly_groundtruth.csv')
```

(I am working on turning this into a standard fixture in #3152; it'll help me do that if we're consistent about the current version)